platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -339,12 +340,16 @@ public void doImportSucceedsWithWrapperRestart() throws Exception {
     MigrationReport report = configurationMigrationManager.doImport(path);
 
     assertThat("Import was not successful", report.wasSuccessful(), is(true));
-    assertThat("Restart system property was set", System.getProperty(RESTART_JVM), nullValue());
+    assertThat(
+        "Restart system property was not cleared",
+        System.getProperty(RESTART_JVM),
+        equalTo("true"));
+    assertThat(ddfBin.resolve("restart.jvm").toFile().exists(), equalTo(false));
```

Comment:
❓Instead of checking for `true` or `false` when we assert things on a file, could we use [Hamcrest `FileMatchers`](http://hamcrest.org/JavaHamcrest/javadoc/2.0.0.0/org/hamcrest/io/FileMatchers.html) instead? So do something like
```
assertThat(ddfBin.resolve("restart.jvm").toFile(), is(not(anExistingFile())));
```

Reply:
🤔 I'll do that one